src/datepicker/datepicker.component.ts

```diff
@@ -201,6 +201,9 @@ export class DatePicker implements OnDestroy, OnChanges, AfterViewChecked, After
 				}
 			}
 		},
+		onDayCreate: (_dObj, _dStr, _fp, dayElem) => {
+			dayElem.className += " bx--date-picker__day";
```

Comment:
We should probably use `classList` with a `contains` check to make sure we don't end up adding the class too many times - https://developer.mozilla.org/en-US/docs/Web/API/Element/classList ... I know flatpickr does some weird things with reusing elements, I'm not sure if this would be the same.

Reply:
you can just do `classList.add()` and it will skip the add if the className already exists

Reply:
Oh sweet, that definitely simplifies that